### PR TITLE
Dump request and response body to logs

### PR DESCRIPTION
**Pull request by dwick**
👓 @tsegaran @spladug 

Having trouble reproducing and issue with the bourne roadblock on staging so we need some more info.  Not seeing anything weird in any of the statsd graphs either.  @spladug is volume going to be an issue here?  We can probably revert this after we figure out what's going on.

### CODE REVIEW

**Comment by tsegaran** [comment]
I think we'll need to sample so as not to overwhelm the log server. 1% should be enough to diagnose frontpage problems...
💅

**Comment by dwick** [comment]
💇  https://github.com/reddit/reddit-private/pull/1396

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -928,6 +928,12 @@ def adzerk_request(keywords, uid, num_placements=1, timeout=1.5,
...
         response = adzerk_api.handle_response(r)
+
+        # Temporarily log request data and response body,
+        # sample at 1%
+        if random.random() < g.live_config.get('ad_log_sample_rate', 0):
+            g.log.info("ad_request: %s, ad_response: %s", json.dumps(data), json.dumps(r))
+
     except adzerk_api.AdzerkError:
         g.stats.simple_event('adzerk.request.badresponse')
         g.log.error('adzerk_request: bad response (%s) %r', r.status_code,
```
Comment (tsegaran) [inline reddit_adzerk/adzerkpromote.py]: Add the request here too, so we see it for failures as well.

**Comment by tsegaran** [comment]
💅

**Comment by dwick** [comment]
💇

**Comment by spladug** [comment]
:fish: but note that you might run into syslog limits with those json responses. individual log entries can only be so large. (8k i think?)

**Comment by dwick** [comment]
trying something else

**Pull request merged**